neurobooth_os/mock/mock_server_acq.py

```diff
@@ -84,7 +84,7 @@ def mock_acq_routine(host, port, conn):
 
             print("Starting recording")
             filename, task = data.split("::")[1:]
-            fname = config.paths["data_out"] + filename
+            fname = config.neurobooth_config["data_out"] + filename
```

Comment:
I know it was this way before, but I personally dislike path construction via concatenation. Would covert to an os.path.join call.

Reply:
Will do